cancel_eur_subscription_auto.py

```
@@ -0,0 +1,137 @@
+    # Get subscription details
+    print(f"\n[1] Checking subscription...")
+    try:
+        sub = stripe.Subscription.retrieve(EUR_SUBSCRIPTION_ID)
+        print(f"  Status: {sub.status}")
+        print(f"  Currency: {sub.currency.upper()}")
+
+        if sub.status == 'active':
+            print(f"\n  ⚠️  This is an ACTIVE EUR subscription")
+            print(f"     It's blocking new USD subscriptions")
+
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)